### PR TITLE
Announce PHP 8.3.8, PHP 8.2.20, and PHP 8.1.29

### PR DESCRIPTION
**Pull request by saundefined**
I suggest editing the message for security releases while we wait for the announcements to be made.

- [x] PHP 8.3.8
- [x] PHP 8.2.20
- [x] PHP 8.1.29

### CODE REVIEW

**Review by ericmann — APPROVED** [review]

File: toots/2024-06-06-php-announce.toot [hunk 1]
```diff
@@ -0,0 +1,18 @@
+📣 Announcing the availability of:
+
+- PHP 8.3.8
+- PHP 8.2.20
+- PHP 8.1.29
+
+‼️ These SECURITY releases fix:
+
+- Argument Injection in PHP-CGI
```
Comment (adoy) [inline toots/2024-06-06-php-announce.toot:9]: Should we add the CVE numbers ?

Comment (saundefined) [reply toots/2024-06-06-php-announce.toot:9]: We have only 500 chars :(

Comment (adoy) [reply toots/2024-06-06-php-announce.toot:9]: Ok all good for me then !

**Review by adoy — APPROVED** [review]

**Comment by derickr** [comment]
Somebody please merge this once the announcements have been sent. I can't as I don't have my security token on me.

**Pull request merged**